docs/api/loading.md

```diff
@@ -32,11 +32,78 @@ Alternatively, developers can configure the loading indicator to dismiss automat
 
 ### Controller
 
-TODO Playground
+import Controller from '@site/static/usage/loading/controller/index.md';
+
+<Controller />
 
 ### Inline
 
-TODO Explainer
+Ionic React and Ionic Vue users also have the option to use the `ion-loading` component directly in their template.
+
+<Tabs defaultValue="react" values={[{ value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+<TabItem value="react">
+
+```tsx
+import React, { useState } from 'react';
+import { IonLoading, IonButton, IonContent } from '@ionic/react';
+
+export const LoadingExample: React.FC = () => {
```

Comment:
When I paste this code into Stackblitz (https://stackblitz.com/edit/angular-xttd3w), it's showing two `ion-loading`s doubled up on each other on first load: https://user-images.githubusercontent.com/90629384/175956610-c092bfe0-ab1d-4a33-8266-4ea5f906ed14.mp4